homeassistant/components/onewire/manifest.json

```diff
@@ -2,7 +2,9 @@
   "domain": "onewire",
   "name": "Onewire",
   "documentation": "https://www.home-assistant.io/integrations/onewire",
-  "requirements": [],
+  "requirements": [
+    "pyownet==0.10.0.post1"
+  ],
   "dependencies": [],
   "codeowners": []
```

Comment:
Do you want to become codeowner of this integration?

Reply:
I guess as the guy who has 3 separate 1-wire networks in his house, it would make sense.  :)  Should I edit that field?  What would go in there?  'garbled1' ?

Reply:
Yes. :+1:

Then run the hassfest script.